ansible/roles/sshkey-host-access/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+LOCAL_PUBKEY_DIR: "/tmp/"
+
+KEY_LOC_PATH: "/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh"
```

Comment:
Use `*_DIR` name to refer to a directory `*_PATH` to refer to a path.. unspoken convention that is inconsistently applied :)